pyproject.toml

```
@@ -49,21 +49,24 @@ dependencies = [
   "pytest-cov>=4.0.0,<5.0.0",
   "black>=23.1.0",
   "ruff>=0.0.243",
-  "isort>=2.5.0",
-  "databricks-connect"
+  "databricks-connect",
+  "types-pyYAML",
+  "types-pytz",
+  "databricks-labs-pylint~=0.1.0",
```

```suggestion
  "databricks-labs-pylint~=0.4.0",
```
we have newer versions :)

---

Updated the version